src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -145,6 +147,15 @@ public String getCommandWord() {
         return COMMAND_WORD;
     }
 
+    @Override
+    public String undo(Model model, CommandHistory pastCommands) {
+        Person bfrEdit = this.getUneditedPerson();
```

Comment:
The abbreviation used in `bfrEdit` is acceptable here since this variable is confined to a small scope. Though it is clearer to spell out the name fully `beforeEdit` for readability sake.